gnc/navigator_msg_multiplexer/nodes/ogrid_arbiter.py

```diff
@@ -139,7 +142,6 @@ def __init__(self, frame_id='enu', map_size=500, resolution=0.3, rate=1):
         self.publisher = rospy.Publisher('/ogrid_master', OccupancyGrid, queue_size=1)
 
         self.ogrid_server = Server(OgridConfig, self.dynamic_cb)
-        dynam_client = Client("bounds_server", config_callback=self.bounds_cb)
```

Comment:
This needs to be here so the callback is registed, just remove the variable = part